lib/orders/submit_trade.js

```diff
@@ -12,12 +12,27 @@ const tradeForOrder = require('./trade_for_order')
  * @return {Object} res - map of order and trade objects
  */
 module.exports = async (state = {}, orderParams = {}) => {
-  const { ws } = state
-  const backtesting = !ws
+  let {
+    ws,
+    backtesting,
+    submitOrder = _submitOrder,
+    liveStream
+  } = state
 
-  const o = !backtesting
-    ? await submitOrder(state, orderParams)
-    : simulateOrderFill(state, orderParams)
+  // backwards compat
+  if (backtesting !== true || backtesting !== false) {
+    backtesting = !ws
+  }
+
+  let o
+  if (liveStream) {
+    await submitOrder(state, orderParams)
```

Comment:
Shouldnt we use `o = await submitOrder(state, orderParams)` if its  a livestream?

Reply:
true, but just if its not a custom function which was always the case for me... hmm... thinking...